src/Evolution/Systems/RelativisticEuler/Valencia/PrimitiveFromConservative.cpp

```diff
@@ -110,11 +111,16 @@ void PrimitiveFromConservative<ThermodynamicDim, Dim>::apply(
       FunctionOfZ<ThermodynamicDim>{tilde_d, tilde_tau, tilde_s_magnitude,
                                     sqrt_det_spatial_metric, equation_of_state};
 
-  const auto z =
-      // NOLINTNEXTLINE(clang-analyzer-core)
-      RootFinder::toms748(f_of_z, lower_bound, upper_bound,
-                          10.0 * std::numeric_limits<double>::epsilon(),
-                          10.0 * std::numeric_limits<double>::epsilon(), 100);
+  DataVector z;
+  try {
+    // NOLINTNEXTLINE(clang-analyzer-core)
+    z = RootFinder::toms748(f_of_z, lower_bound, upper_bound,
+                            10.0 * std::numeric_limits<double>::epsilon(),
+                            10.0 * std::numeric_limits<double>::epsilon(), 100);
+  } catch (std::exception& exception) {
+    ERROR("Failed to find z with TOMS748 root finder. Got exception message:\n"
```

Comment:
Not a necessary change, just a suggestion I'd like your thoughts on :) 

How about `Failed to find the intermediate variable z with TOMS748 root finder while computing the primitive variables from the conserved variables. Got exception...`?

Reply:
I incorporated your suggestion. I squashed the change.